### DISCUSSION TITLE
mockoon server Unknown hostname/address displayed with no solution

### DISCUSSION THREAD

**Discussion by jessewriter** [mockoon/mockoon#1232] [Help]
I set the address to match the client device I want to mock for example 10.0.0.44:8080 and I get the following error when restarting the service server Unknown hostname/address provided under the settings tab in the gui mockoon app.

I restarted the server with same error.

https://mockoon.com/docs/latest/server-configuration/listening-hostname/ document makes it sound like you can set any ip address you choose but that is not the case and there are no instructions on how to fix this situation.  I'm not a network expert but I'm thinking I need to change my network adapter settings or hosts file??

as a user please provide documentation how to be successful with this app and changing the hostname.

environment: windows 11, mockoon 6.0.1

<img width="328" alt="image" src="https://github.com/mockoon/mockoon/assets/16365134/7c4ce1b1-f9f7-43e4-89c1-50207bee0d4c">

**Comment by 255kb** [comment]
The hostname must be a local (internal) IP address on the machine Mockoon is running on. It will listen to the specified port on this specific IP. 
It looks like you are using Windows. To list the available local IP addresses you can use the `ipconfig` command:

```
$ ipconfig

Windows IP Configuration

Ethernet adapter Ethernet:

   Connection-specific DNS Suffix  . : lan
   IPv4 Address. . . . . . . . . . . : 192.168.86.112 <-- IP you can use
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : 192.168.86.1
```

In general, you can listen to IP+port like `127.0.0.1`, `localhost`, `192.168.x.x`. There may be more of them if you have multiple network adapters. 
But you won't be able to listen to external IPs, or third parties IPs. 

What you need to do:
- run mockoon on localhost and port 8080.
- in the app consuming the original API, replace `10.0.0.44:8080` with `localhost:8080`.
- if you want to only mock some endpoints, and forward the rest of the requests to your original API, exanle the proxy mode and set the proxy URL to `http://10.0.0.44:8080`.

![image](https://github.com/mockoon/mockoon/assets/7489814/d4a09d93-7df0-4a14-9575-6f256871d82d)
